recipe-server/client/control/components/recipes/RecipeDetails.js

```diff
@@ -104,14 +104,29 @@ class ArgumentsValue extends React.PureComponent {
   render() {
     const { name, value } = this.props;
 
+    let inner;
+
     if (name === 'branches') {
-      return this.renderBranchTable(value);
+      inner = this.renderBranchTable(value);
+    } else if (name === 'extra_filter_expression') {
+      inner = <pre><code>{value}</code></pre>;
+    } else if (typeof value === 'boolean') {
+      inner = value ? 'True' : 'False';
```

Comment:
I think it'd be a good idea to abstract out these `inner = ...` bits, like we have for `renderBranchTable`. This would allow us to write a bit more granular tests against the `ArgumentsValue` component, plus make the `render` function a bit more concise.